resources/user-agents/apps/facebook/facebook-app.json

```diff
@@ -456,7 +455,13 @@
             "XT1080": "Motorola XT1080",
             "XT907": "Motorola XT907",
             "Z793C": "ZTE Z793C",
-            "Z970": "ZTE Z970"
+            "Z970": "ZTE Z970",
+            "GT-N5110": "Samsung GT-N5110",
```

Comment:
As you may have noticed in some of my recent PRs, I've been trying to alphabetize these device blocks (by device match text. I just highlight the block in Sublime and press F5), as it makes it easier to quickly scan the list to see if a device's match exists already or not.

Any objections in doing this moving forward?  I think it would be nice to eventually have a sort of cleanup tool that does these sorts of things automatically (unless you have something already).